apps/web/src/app/api/events/[id]/cancel/route.ts

```
@@ -0,0 +1,32 @@
+      return NextResponse.json({ error: 'Event ID required' }, { status: 400 });
+    }
+
+    // 1. Mark event as cancelled in DB
+    // await db.event.update({ where: { id: eventId }, data: { status: 'CANCELLED' } })
+
+    // 2. Queue cancellation for async processing (to prevent long HTTP request)
+    await queue.pushToQueue(eventId);
+
+    return NextResponse.json({
+      success: true,
+      message: 'Event cancelled. Refunds queued for processing.'
```

ESLint

[error] Replace `'Event·cancelled.·Refunds·queued·for·processing.'` with `"Event·cancelled.·Refunds·queued·for·processing.",`

(prettier/prettier)